xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -271,6 +271,7 @@ void CGUIWindowVideoBase::OnItemInfo(CFileItem* pItem, ADDON::ScraperPtr& scrape
     Refresh();
     m_viewControl.SetSelectedItem(itemNumber);
   }
+  return modified;
```

Comment:
I don't think this works. If the user simply opens and closes the info dialog, `modified` will be `false` which will result in the video being automatically played after having opened and closed the info dialog.